src/chromeDebugAdapter.ts

```diff
@@ -79,7 +83,8 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
                 runtimeExecutable = re;
             }
 
-            runtimeExecutable = runtimeExecutable || utils.getBrowserPath(args['version']);
+            // Use runtimeExecutable if it is set; if not, check for version flag; else launch stable version of Microsoft Edge Chromium
+            runtimeExecutable = runtimeExecutable || utils.getBrowserPath(args['version']) || utils.getBrowserPath('stable');
```

Comment:
Could make this an else clause off of `if (args.runtimeExecutable)` to make it more clear that we are using the version/stable path if the user didn't pass the runtimeExectuable arg.

Reply:
Yep!